Makefile

```diff
@@ -65,6 +66,24 @@ $(WINDOWS_AMD64_BINARY): $(SOURCES)
 	./scripts/build_variant.sh windows amd64 $(VERSION) $(GITCOMMIT_SHA)
 	@mv ./bin/windows-amd64/$(BINARY_NAME) ./$(WINDOWS_AMD64_BINARY)
 
+GITCOMMIT_SHA: $(GITFILES)
+	git rev-parse --short HEAD > GITCOMMIT_SHA
```

Comment:
It would be wise to specify a length to `--short` so that no git or distribution modifications cause this to change length.

Reply:
I can if you'd like; I don't have a strong opinion here.  I'm not using this SHA for anything other than an indication as to the source from which the binary was built (debugging); there is no programmatic dependency on it or its length right now.

Any particular suggestion on the length?

Reply:
7 :+1: 	

I don't see any distributed binaries so there's not really a notion of a "known length". 7 is the default according to `man git-rev-parse`, so that seems like a sane constant to provide. Additionally, Github looks to be using 7 char SHAs when there are short SHA links.